source/android/Renaissance/lib/models/path.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:portakal/models/basic_path.dart';
 import 'package:portakal/models/milestone.dart';
 
-class Path {
+class Pathh {
```

Comment:
It looks like there is a typo in the name of the path class. You should fix it first Kürşat.

Reply:
I actually kinda like it.